generated/nifake/nifake/session.py

```diff
@@ -1357,7 +1357,7 @@ def one_input_function(self, a_number):
         self._interpreter.one_input_function(a_number)
 
     @ivi_synchronized
-    def parameters_are_multiple_types(self, a_boolean, an_int32, an_int64, an_int_enum, a_float, a_float_enum, a_string):
+    def parameters_are_multiple_types(self, a_boolean, an_int32, an_int64, an_int_enum, a_float, a_float_enum, string_size, a_string):
```

Comment:
@marcoskirsch The `string_size` parameter got added because the `a_string` parameter used to have an `array` section that referenced `string_size` as its size but that `array` section was [removed](https://dev.azure.com/ni/DevCentral/_git/ni-central/pullrequest/386190).

Should the `string_size` parameter be removed now, too? Should it stay, but have a calculated value? Was the removal of the `array` section incorrect??

Are there any MI driver APIs that accept a non-null-terminated string? If so, how would those be configured in the metadata?

Reply:
Alright, then I think it's inappropriate to still have the `string_size` parameter. Tom, please remove it from the metadata, and make whatever test updates are necessary (like not expecting `string_size` to be passed to the driver, if we have a test for that).